repm/repm.go

```diff
@@ -47,9 +47,9 @@ type Logger interface {
 // is created. Logger receives logging output from Replicache.
 func Init(storageDir, tempDir string, logger Logger) {
 	if logger == nil {
-		zlog.Logger = zlog.Output(zl.ConsoleWriter{Out: os.Stdout})
+		zlog.Logger = zlog.Output(zl.ConsoleWriter{Out: os.Stdout, TimeFormat: "02 Jan 06 15:04:05.000 -0700"})
```

Comment:
wedding day?

Reply:
Kinda high resolution for that though.

Reply:
[fyi] That date is part of the API. https://golang.org/pkg/time/#Time.Format

Reply:
the thing that makes this comment funny for me is that @aboodman i think we have the same wedding day, which is why I think you couldn't come to mine...

Reply:
lol, whoopsie